### ISSUE TITLE
Dynamic World training and test set

### ISSUE THREAD

**Issue by hannah-rae** [nasaharvest/cropharvest#121]
From their [paper](https://www.nature.com/articles/s41597-022-01307-4#Sec8): 

> Our 409-tile test dataset, including expert consensus annotations and corresponding Dynamic World estimated probabilities and class labels for each 5100 m × 5100 m tile are archived in Zenodo at the following https://doi.org/10.5281/zenodo.4766508[36](https://www.nature.com/articles/s41597-022-01307-4#ref-CR36). The training dataset has been archived in PANGAEA in a separate repository: https://doi.org/10.1594/PANGAEA.933475[37](https://www.nature.com/articles/s41597-022-01307-4#ref-CR37). The training and test data collected for Dynamic World are also available as Earth Engine Image Collection and can be accessed with: `ee.ImageCollection(‘projects/wri-datalab/dynamic_world/v1/DW_LABELS’)`
